### DISCUSSION TITLE
Live Stream PubSub Notifications

### DISCUSSION THREAD

**Discussion by cyrillicw** [uNetworking/uWebSockets#1220] [Q&A]
Can I use the library in order to notify viewers of my live stream that the new video segment is ready for download?

**Comment by ghost** [comment]
No probably not. You probably want to send an email for that.

**Comment by cyrillicw** [comment]
@alexhultman Maybe I put it wrong. I have segments of live stream, each 1 second length. I have a p2p network in which these segments are shared. As soon as a new such segment is generated I want to notify all py peers that they can download it

**Comment by Zentropivity** [comment]
@cyrillicw Look:
https://github.com/uNetworking/uWebSockets/issues/1048
